appscale/tools/appscale.py

```diff
@@ -755,6 +735,19 @@ def destroy(self):
     command = []
     contents_as_yaml = yaml.safe_load(contents)
 
+    if 'verbose' in contents_as_yaml and contents_as_yaml['verbose'] == True:
+      is_verbose = contents_as_yaml['verbose']
+      command.append("--verbose")
+    else:
+      is_verbose = False
+
+    if 'keyname' in contents_as_yaml:
+      keyname = contents_as_yaml['keyname']
+      command.append("--keyname")
+      command.append(contents_as_yaml['keyname'])
+    else:
+      keyname = 'appscale'
```

Comment:
Is this behavior done elsewhere? It would make more sense to me to raise an exception, but I might be missing something.